libevent-sys/Cargo.toml

```diff
@@ -15,10 +15,11 @@ links = "event"
 build = "build.rs"
 
 [features]
-default = [ "pkgconfig", "openssl", "threading" ]
+default = [ "pkgconfig", "openssl", "threading", "buildtime_bindgen" ]
 static = []
 pkgconfig = [ "pkg-config" ]
 bundled = [ "static", "cmake" ]
+buildtime_bindgen = [ "pkgconfig" ]
```

Comment:
Bindgen should switch independently of `pkgconfig`/`bundled`, unless I'm missing something? And the `bindgen` build dep can now be optional, enabled by this feature.